pkg/halyard/validators.go

```diff
@@ -0,0 +1,173 @@
+package halyard
+
+import "github.com/armory/spinnaker-operator/pkg/apis/spinnaker/v1alpha2"
+
+// Validators we're going always going to skip
```

Comment:
Can you document that all validator names should exactly match the class names in Halyard extending `Validator` class?
Also this seems like a fragile solution, ideally it should be possible to just run all validators without actually naming them, otherwise we would need to constantly update this file every time someone adds/deletes or renames a validator class in Halyard, this could get out of sync pretty quickly.

Reply:
Here we just list the validations that you do NOT want to run (those the operator implements itself).

Reply:
I know that, but it should be commented in the code, there's no way someone without the context will know that validator names should match exactly Java class names in Halyard

Reply:
I can add a comment that this should match the Halyard class name. I was confused by the other part of the comment. Yes if validation class names change, they'd not be skipped. But I don't recall that happening and the operator is tied to Halyard for now anyway. When we implement all validations in the operator, we won't be calling out to Halyard anymore.

Reply:
Added comment